### PR TITLE
Usernames

### PR DESCRIPTION
**Pull request by Oransj**

### CODE REVIEW

File: cmd/username_populator/main.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	"os"
+
+	"github.com/Molnes/Nyhetsjeger/internal/database"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("DB Populator: Error loading .env")
```
Comment (Molnes) [inline cmd/username_populator/main.go:16]: should not be a fatal error, since the environment isn't necessarily in the .env

**Review by z-rayc — DISMISSED** [review]

**Review by Molnes — CHANGES_REQUESTED** [review]

File: cmd/username_populator/main.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	// Populate usernames with raudlefse, fintaco, brennandeand, raudstol, and finappelsin.
+	ctx := context.Background()
+	tx, err := db.BeginTx(ctx, nil)
+	if err != nil {
+		log.Println(err)
+	}
+
+	tx.Exec("INSERT INTO usernames VALUES ('raud', 'lefse')")
+
+	if err := tx.Commit(); err != nil {
```
Comment (Molnes) [inline cmd/username_populator/main.go:85]: add comment memember

**Review by Molnes — CHANGES_REQUESTED** [review]

File: cmd/username_populator/main.go (hunk 1 above)
Comment (Oransj) [reply cmd/username_populator/main.go:16]: fixed

File: cmd/username_populator/main.go (hunk 2 above)
Comment (Oransj) [reply cmd/username_populator/main.go:87]: fixed

**Pull request merged**